ci/jenkins/Jenkinsfile_unix_cpu

```diff
@@ -21,7 +21,7 @@
 // See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
 
 // timeout in minutes
-max_time = 180
+max_time = 240
```

Comment:
why wasn't this discussed before? Sneaking this change here is not ok, this should go in a separate PR and clearly marked as such.